be/src/olap/rowset/alpha_rowset_writer.cpp

```diff
@@ -116,6 +114,10 @@ OLAPStatus AlphaRowsetWriter::add_rowset(RowsetSharedPtr rowset) {
         RETURN_NOT_OK(flush());
         _num_rows_written += segment_group->num_rows();
     }
+    // process delete predicate
+    if (rowset->rowset_meta()->has_delete_predicate()) {
+        _current_rowset_meta->set_delete_predicate(rowset->rowset_meta()->delete_predicate());
```

Comment:
add_delete_predicate?

Reply:
delete predicate is not a list or a array. I think set is better.

Reply:
delete predicate is not a list or array. I think set is better